app/components/header.tsx

```diff
@@ -0,0 +1,27 @@
+import * as NavigationMenu from "@radix-ui/react-navigation-menu";
+
+export const Header = () => {
```

Comment:
no es importante como se _declara_ un componente, pero si es importante ser consistente. Te propongo que declaremos todas las funciones que estan en el root del file usemos la `function` keyword. Estar _en el root del file_ significa que no esta dentro de ninguna otra funcion



```suggestion
export function Header() {
```